tests/Hedgehog.Tests/GenTests.fs

```diff
@@ -50,19 +70,21 @@ let genTests = testList "Gen tests" [
             |> (fun x -> x.Split ([|System.Environment.NewLine|], System.StringSplitOptions.None))
             |> Array.item 1
             |> System.DateTime.Parse
-        System.DateTime (2000, 1, 1) =! result
 
-    testCaseNoFable "int64 can create exponentially bounded integer" <| fun _ ->
+        System.DateTime (2000, 1, 1) =! actual
```

Comment:
This value is repeated a bit, can we make a binding for it?

Reply:
It's just in 2 places so I left it as-is.